domain/application/state/application.go

```diff
@@ -1549,6 +1549,10 @@ WHERE uuid = $applicationID.uuid
 			return errors.Errorf("refreshing application config: %w", err)
 		}
 
+		if err := st.refreshApplicationEndpointBindings(ctx, tx, appIdent, charmIdent); err != nil {
```

Comment:
nit:

```suggestion
		if err := st.updateApplicationEndpointBindings(ctx, tx, appIdent, charmIdent); err != nil {
```

Reply:
I'm not sure if I agree, in other places we use 'update' when binding to a new space. However, here we are updating the charm relations